pkg/errors/error_message.go

```diff
@@ -38,6 +38,7 @@ const (
 	// kafka topic commands
 	FailedToCreateProducerErrorMsg    = "failed to create producer: %v"
 	FailedToCreateConsumerErrorMsg    = "failed to create consumer: %v"
+	FailedToGetConfigurationErrorMsg  = "failed to get configuration map: %v"
```

Comment:
Let's use `%w` to wrap errors. This lets us unwrap them later if we need to!
```suggestion
	FailedToGetConfigurationErrorMsg  = "failed to get configuration map: %w"
```